aviary/variable_info/enums.py

```diff
@@ -181,3 +181,27 @@ class Flap_Type(Enum):
     """
     Double-slotted Fowler flaps
     """
+
+
+@unique
+class Reserves_Type(Enum):
```

Comment:
This flag shouldn't be necessary - check out the FLOPS mass scalars for an example of how we've split variables in a way that doesn't require a user-set flag.

Reply:
This class is removed.